server/src/main/java/com/paypal/selion/grid/SeLionGridLauncher.java

```diff
@@ -96,11 +176,19 @@ public void shutdown() throws Exception {
      */
     public void boot(String[] args) throws Exception {
         CommandLineOptionHelper helper = new CommandLineOptionHelper(args);
+        GridRole role = GridRole.find(args);
+
+        if (role == null) {
+            printInfoAboutRoles(helper);
+            return;
```

Comment:
`GridRole.find()` defaults to `GridRole#NOT_GRID` if `-role` is not specified for us but can also return `null` on errors. Thus, the reason for this check and the default message.  For example, `-role astronaut` will trigger the `find` to return `null`